src/components/Title.js

```diff
@@ -0,0 +1,16 @@
+import selectedUserStore, { GET_USER } from '../modules/selectedUser.js';
+
+const Title = () => {
+  const $title = document.querySelector('h1#user-title');
```

Comment:
```suggestion
  const $title = document.querySelector('#user-title');
```

CSS id속성의 경우 한 문서에 고유한 값 하나만 존재할 수 있기 때문에 위 코드처럼 아이디만 명시해서 바로 파싱해오곤 했는데 태그 이름을 명시해주면 해당 DOM 요소가 어떤 태그인지도 알 수 있겠네요! 좋은 방법인거 같습니다👍👍👍